tests/system/test_system.py

```diff
@@ -40,12 +41,21 @@
 MISSING_DOCUMENT = "No document to update: "
 DOCUMENT_EXISTS = "Document already exists: "
 UNIQUE_RESOURCE_ID = unique_resource_id("-")
+FIRESTORE_EMULATOR_HOST = "FIRESTORE_EMULATOR_HOST"
```

Comment:
I don't think we need this constant. It's used once. As it's name completely repeats the value itself, there is no practical use in it.

Reply:
:+1:

Reply:
It is need as chris also used here, the pattern that they follow is one is for internal variable and one is for variable which user use like [here  also ](https://github.com/googleapis/python-cloud-core/blob/master/google/cloud/environment_vars.py) https://github.com/q-logic/python-firestore/blob/160d7d11564a137f06faced96c21f967411bfc55/google/cloud/firestore_v1/client.py#L57

I am trying to use the same which already declared by chris.